docs/scripts_md/run_defacing_script.md

```diff
@@ -0,0 +1,136 @@
+# NAME
+
+run\_defacing.pl -- a script that creates defaced images for anatomical
+acquisitions specified in the Config module of LORIS.
+
+# SYNOPSIS
+
+perl tools/run\_defacing\_script.pl `[options]`
+
+Available options are:
+
+\-profile     : name of the config file in `../dicom-archive/.loris_mri`
+\-tarchive\_ids: comma-separated list of MySQL TarchiveIDs
+\-verbose     : be verbose
+
+# DESCRIPTION
+
+This script will create defaced images for anatomical acquisitions that are
+specified in the Config module of LORIS.
+
+# METHODS
+
+### grep\_FileIDs\_to\_deface($session\_id\_arr, $modalities\_to\_deface\_arr)
+
+Queries the database for the list of acquisitions' FileID to be used to run the
+defacing algorithm based on the provided list of SessionID and Scan\_type to
+restrict the search.
+
+INPUTS:
+  - $session\_id\_arr          : array of SessionIDs to use when grepping FileIDs
+  - $modalities\_to\_deface\_arr: array of Scan\_type to use when grepping FileIDs
+
+RETURNS: hash of matching FileIDs to be used to run the defacing algorithm
+         organized in a hash as follows:
+            {0123}                      # sessionID key
+              {flair}                   # flair scan type key
+                {$FileID} = $File\_path  # key = FileID; value = MINC file path
+              {t1}                      # t1 scan type key
+                {$FileID} = $File\_path  # key = FileID 1; value = MINC file 1 path
+                {$FileID} = $File\_path  # key = FileID 2; value = MINC file 2 path
```

Comment:
doesn't render well, all on the same line. suggest using triple single quote for line 35-40
\`\`\`
text here
\`\`\`

Reply:
it renders better in my next commit but it is not perfect. 

I can't find a way to have multiline code ``` automatically set in the markdown when running pod2markdown. We'll try to find a better solution with @nicolasbrossard for that.

Reply:
done!